packages/tools/devtools/devtools-core/api-report/devtools-core.alpha.api.md

```diff
@@ -8,12 +8,14 @@
 export interface ContainerDevtoolsProps extends HasContainerKey {
     container: IContainer;
     containerData?: Record<string, IFluidLoadable>;
-    isDataObject?: boolean;
 }
 
 // @beta
 export type ContainerKey = string;
 
+// @alpha
```

Comment:
I don't think we need either of these new APIs. `registerContainerDevtools` can continue to take `IContainer`, and `registerDataObject` can take `IPureDataObject`. I don't think we should need anything new in the API except the `registerDataObject`.